---------------------------------------
Begin Slurm Prolog: Mar-27-2025 15:32:32
Job ID:    2379913
User ID:   mtalreja6
Account:   coc
Job name:  LayerSkipWithLayerDrop
Partition: ice-gpu
QOS:       coc-ice
---------------------------------------
/var/lib/slurm/slurmd/job2379913/slurm_script: line 14: huggingface-cli: command not found
/usr/local/pace-apps/manual/packages/anaconda3/2023.03/bin/python
[W socket.cpp:464] [c10d] The server socket has failed to listen on [::]:29500 (errno: 98 - Address already in use).
[W socket.cpp:464] [c10d] The server socket has failed to listen on [::]:29500 (errno: 98 - Address already in use).
[W socket.cpp:464] [c10d] The server socket has failed to listen on [::]:29500 (errno: 98 - Address already in use).
[W socket.cpp:464] [c10d] The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
[E socket.cpp:500] [c10d] The server socket has failed to listen on any local network address.
[W socket.cpp:464] [c10d] The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
[E socket.cpp:500] [c10d] The server socket has failed to listen on any local network address.
[W socket.cpp:464] [c10d] The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
[E socket.cpp:500] [c10d] The server socket has failed to listen on any local network address.
Traceback (most recent call last):
  File "/storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/layer_skip_env/bin/torchrun", line 8, in <module>
Traceback (most recent call last):
  File "/storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/layer_skip_env/bin/torchrun", line 8, in <module>
Traceback (most recent call last):
  File "/storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/layer_skip_env/bin/torchrun", line 8, in <module>
    sys.exit(main())
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 347, in wrapper
    sys.exit(main())
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 347, in wrapper
    sys.exit(main())
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 347, in wrapper
    return f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 812, in main
    return f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 812, in main
    return f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 812, in main
    run(args)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 803, in run
    run(args)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 803, in run
    run(args)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 803, in run
    elastic_launch(
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    elastic_launch(
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    elastic_launch(
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 259, in launch_agent
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 259, in launch_agent
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 259, in launch_agent
    result = agent.run()
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = agent.run()
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 727, in run
    result = agent.run()
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 727, in run
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 727, in run
    result = self._invoke_run(role)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 862, in _invoke_run
    result = self._invoke_run(role)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 862, in _invoke_run
    result = self._invoke_run(role)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 862, in _invoke_run
    self._initialize_workers(self._worker_group)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    self._initialize_workers(self._worker_group)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 699, in _initialize_workers
    self._initialize_workers(self._worker_group)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 699, in _initialize_workers
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 699, in _initialize_workers
    self._rendezvous(worker_group)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    self._rendezvous(worker_group)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    self._rendezvous(worker_group)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 123, in wrapper
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 542, in _rendezvous
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 542, in _rendezvous
    result = f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 542, in _rendezvous
    store, group_rank, group_world_size = spec.rdzv_handler.next_rendezvous()
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/rendezvous/static_tcp_rendezvous.py", line 55, in next_rendezvous
    store, group_rank, group_world_size = spec.rdzv_handler.next_rendezvous()
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/rendezvous/static_tcp_rendezvous.py", line 55, in next_rendezvous
    store, group_rank, group_world_size = spec.rdzv_handler.next_rendezvous()
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/rendezvous/static_tcp_rendezvous.py", line 55, in next_rendezvous
    self._store = TCPStore(  # type: ignore[call-arg]
torch.distributed.DistNetworkError: The server socket has failed to listen on any local network address. The server socket has failed to listen on [::]:29500 (errno: 98 - Address already in use). The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
    self._store = TCPStore(  # type: ignore[call-arg]
torch.distributed.DistNetworkError: The server socket has failed to listen on any local network address. The server socket has failed to listen on [::]:29500 (errno: 98 - Address already in use). The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
    self._store = TCPStore(  # type: ignore[call-arg]
torch.distributed.DistNetworkError: The server socket has failed to listen on any local network address. The server socket has failed to listen on [::]:29500 (errno: 98 - Address already in use). The server socket has failed to bind to 0.0.0.0:29500 (errno: 98 - Address already in use).
srun: error: atl1-1-03-013-13-0: tasks 0-2: Exited with exit code 1
Error importing huggingface_hub.hf_api: No module named 'urllib3'
Traceback (most recent call last):
  File "/storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/eval.py", line 12, in <module>
    import transformers
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/transformers/__init__.py", line 26, in <module>
    from . import dependency_versions_check
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/transformers/dependency_versions_check.py", line 16, in <module>
    from .utils.versions import require_version, require_version_core
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/transformers/utils/__init__.py", line 21, in <module>
    from huggingface_hub import get_full_repo_name  # for backward compatibility
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/huggingface_hub/__init__.py", line 962, in __getattr__
    submod = importlib.import_module(submod_path)
  File "/storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/layer_skip_env/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/huggingface_hub/hf_api.py", line 46, in <module>
    import requests
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
ModuleNotFoundError: No module named 'urllib3'
[2025-03-27 15:32:44,972] torch.distributed.elastic.multiprocessing.api: [ERROR] failed (exitcode: 1) local_rank: 0 (pid: 1527820) of binary: /storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/layer_skip_env/bin/python
Traceback (most recent call last):
  File "/storage/ice1/4/0/mtalreja6/layerskip/LayerSkip/layer_skip_env/bin/torchrun", line 8, in <module>
    sys.exit(main())
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 347, in wrapper
    return f(*args, **kwargs)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 812, in main
    run(args)
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/run.py", line 803, in run
    elastic_launch(
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/hice1/mtalreja6/.local/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 268, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
eval.py FAILED
------------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-03-27_15:32:44
  host      : atl1-1-03-013-13-0.pace.gatech.edu
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 1527820)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
srun: error: atl1-1-03-013-13-0: task 3: Exited with exit code 1
---------------------------------------
Begin Slurm Epilog: Mar-27-2025 15:32:45
Job ID:        2379913
User ID:       mtalreja6
Account:       coc
Job name:      LayerSkipWithLayerDrop
Resources:     cpu=4,gres/gpu:h100=1,mem=30G,node=1
Rsrc Used:     cput=00:00:52,vmem=0,walltime=00:00:13,mem=40360K,energy_used=0
Partition:     ice-gpu
QOS:           coc-ice
Nodes:         atl1-1-03-013-13-0
---------------------------------------
